src/Web/Internal/FormUrlEncoded.hs

```diff
@@ -11,6 +12,7 @@
 {-# LANGUAGE OverloadedStrings          #-}
 {-# LANGUAGE PolyKinds                  #-}
 {-# LANGUAGE ScopedTypeVariables        #-}
+{-# LANGUAGE StandaloneDeriving         #-}
```

Comment:
Cannot use, see how `Tagged` instance is written.

Reply:
Happy to do it that way, but I'm curious; why must `Tagged` be written this way?

```haskell
instance ToHttpApiData a => ToHttpApiData (Tagged b a) where
  toUrlPiece        = coerce (toUrlPiece :: a -> Text)
  toHeader          = coerce (toHeader :: a -> ByteString)
  toQueryParam      = coerce (toQueryParam :: a -> Text)
  toEncodedUrlPiece = coerce (toEncodedUrlPiece ::  a -> BS.Builder)
```

Reply:
because `DerivingStrategies` is GHC-8.2+ feature, and I don't find the convenience to justify dropping support for older GHCs. You'll see CI failing soon.

Reply:
Updated with a new commit.